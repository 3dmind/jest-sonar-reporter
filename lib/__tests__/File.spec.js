'use strict'

const xml = require('xml')
const File = require('../File')

describe('File', () => {
  test('<file path=""></file>', () => {
    const mock = {
      testFilePath: 'test/FooTest.js'
    }
    const file = new File(mock)

    const actualReport = xml(file)

    expect(actualReport).toMatchSnapshot()
  })
})
