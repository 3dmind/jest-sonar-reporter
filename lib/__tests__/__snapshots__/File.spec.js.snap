// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File <file path=""></file> 1`] = `"<file path=\\"test/FooTest.js\\"></file>"`;
